src/controller.rs

```diff
@@ -124,12 +123,18 @@ impl MetricsController {
         }
     }
 
+    pub fn start(&self) {
+        self.logger.init();
+    }
+
```

Comment:
Is there somethig that can crash out and prevent us from initializing in the constructor?